repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/pooling/SimpleResourceContext.java

```diff
@@ -19,6 +19,7 @@
  */
 package org.openrepose.commons.utils.pooling;
 
+@FunctionalInterface
 public interface SimpleResourceContext<R> {
 
     void perform(R resource) throws ResourceContextException;
```

Comment:
Feels related to the other, and i still have no idea what they are for, so i have no idea whether or not they should be marked functional.